_posts/2021-08-10-ppa_news.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title: Official PPA revived
+author: Michael Kogan
+permalink: /ppa-revival
```

Comment:
Why do you need the permalink for it? The default one would contain the date, which is more appropriate for news item, I think

Reply:
Right, updated tags and removed permalink!